lib/mayaUsd/render/mayaToHydra/renderGlobals.cpp

```diff
@@ -312,9 +301,6 @@ MObject MtohCreateRenderGlobals() {
         });
     static const TfTokenVector selectionOverlays{MtohTokens->UseHdSt,
                                                  MtohTokens->UseVp2};
```

Comment:
This static var - and the MtohTokens it uses - should probably also be removed. I've pushed a commit that you can add to this PR to do that:

https://github.com/LumaPictures/maya-usd/commit/b99dea4ab927c8fa2b900e821eaec62a1d052508

...or I can make a new PR after this one, whatever you'd prefer.